templates/staff/user_create.html

```diff
@@ -0,0 +1,176 @@
+{% extends "staff/base.html" %}
+
+{% load static %}
+
+{% block extra_styles %}
+<link rel="stylesheet" href="{% static 'vendor/select2.min.css' %}">
+<link rel="stylesheet" href="{% static 'vendor/select2-bootstrap4.min.css' %}">
+{% endblock %}
+
+{% block metatitle %}Create a user: Staff Area | OpenSAFELY Jobs{% endblock metatitle %}
+
+{% block breadcrumbs %}
+<nav class="breadcrumb-container breadcrumb--danger" aria-label="breadcrumb">
+  <div class="container">
+    <ol class="breadcrumb rounded-0 mb-0 px-0">
+      <li class="breadcrumb-item">
+        <a href="{% url 'staff:index' %}">Staff area</a>
+      </li>
+      <li class="breadcrumb-item">
+        <a href="{% url 'staff:user-list' %}">Users</a>
+      </li>
+      <li class="breadcrumb-item active" aria-current="page">
+        Create a user
+      </li>
+    </ol>
+  </div>
+</nav>
+{% endblock breadcrumbs %}
+
+{% block jumbotron %}
+<div class="jumbotron jumbotron-fluid jumbotron--danger pt-md-2">
+  <div class="container">
+    <h1 class="display-4">Create interactive user</h1>
+  </div>
+</div>
+{% endblock jumbotron %}
+
+{% block staff_content %}
+<div class="container">
+  <div class="row">
+    <div class="col-lg-9 col-xl-8">
+      <h2 class="sr-only">Create a Project</h2>
+
+      <p>
+        Creating a user here will set up an account for a user who can only use
+        the Interactive portion of the site.  Please ensure users are part of an
+        <strong>approved application</strong>.
+      </p>
+
+      <form class="mt-3 pt-3" method="POST">
+        {% csrf_token %}
+
+        {% if form.non_field_errors %}
+          <ul>
+            {% for error in form.non_field_errors %}
+              <li class="text-danger">{{ error }}</li>
+            {% endfor %}
+          </ul>
+        {% endif %}
+
+        <fieldset class="mb-3">
+          <legend class="h3 mb-3">Set the user's details</legend>
+
+          {% include "components/form_text.html" with field=form.application_url label="Application URL" name="application_url" %}
```

Comment:
Just had a thought about this... is there any need to have the application url on the create user form? It might get quite annoying when adding several users and there's more potential for error. Given that it should be set per project, should it actually just be a field on the create project modal instead?

Reply:
Are we intending to add multiple Interactive users to a single project?  I was under the impression that it would effectively be 1 user/application, but I might just have that wrong in my head!

Reply:
Ah good point. I'd assumed that there might be more than one now that we have a more complicated application process.